.github/workflows/workflow.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: Build
+# yamllint disable-line rule:truthy
+on:
+  push:
+  pull_request:
+
+jobs:
+
+  list-scenarios:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.listscenarios.outputs.scenarios }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: listscenarios
+        uses: manics/action-ansible-molecule-list-scenarios@main
+
+  test:
+    name: Test
+    needs:
+      - list-scenarios
+    runs-on: ubuntu-latest
+    strategy:
+      # Keep running so we can see if other tests pass
+      fail-fast: false
+      matrix:
+        scenario: ${{fromJson(needs.list-scenarios.outputs.matrix)}}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: "${{ github.repository }}"
+      - name: molecule
+        uses: manics/action-ome-molecule@master
```

Comment:
Very limited knowledge about creating/managing actions, is it preferable to have a simple action wrapping the `ome-ansible-molecule` Docker image or could the action be managed in `ome/ome-ansible-molecule` itself?

Reply:
I think either can work. An action is cloned into the repo that it's used in, `ome-ansible-molecule` is very small so that should be fine. I'll try adding `action.yml` to https://github.com/ome/ome-ansible-molecule/pull/24 and we can see.